### PR TITLE
Add version information

### LINKED ISSUE #4: Add version files

### PR DESCRIPTION
**Pull request by cloudxxx8**
fix #4

Signed-off-by: Cloud Tsai <cloudxxx8@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 GO=CGO_ENABLED=0 go
-GOFLAGS=-ldflags
 
 MICROSERVICES=cmd/device-random
 
 .PHONY: $(MICROSERVICES)
 
+VERSION=$(shell cat ./VERSION)
+
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/devoce-random.Version=$(VERSION)"
```
Comment (mhall119) [inline Makefile]: This looks like a typo, "devoce" should be "device"

**Review by mhall119 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (cloudxxx8) [reply Makefile]: @mhall119 I appreciate your review.  That's really my typo. I've corrected it

Comment (cloudxxx8) [reply Makefile]: @mhall119 please help review my change of this PR

**Review by jpwhitemn — APPROVED** [review]

**Review by mhall119 — APPROVED** [review]
Looks good to me

**Pull request merged**